pdns/comfun.cc

```diff
@@ -198,8 +199,10 @@ struct SendReceiveRes
   {
     d_socket = socket(AF_INET, SOCK_DGRAM, 0);
     int val=1;
-    setsockopt(d_socket, SOL_SOCKET, SO_REUSEADDR, &val, sizeof(val));
-    connect(d_socket, (struct sockaddr*)&remote, remote.getSocklen());
+    if (setsockopt(d_socket, SOL_SOCKET, SO_REUSEADDR, &val, sizeof(val)) != 0)
+       g_log<<Logger::Warning<<"Failed to turn on SO_REUSEADDR: " << stringerror() << endl;
+    if (connect(d_socket, (struct sockaddr*)&remote, remote.getSocklen()) != 0)
```

Comment:
A connect failure feels fatal to me

Reply:
This does not look resolved.

Reply:
so - what should I do here? raise exception?